notebooks/01-variables.ipynb

```diff
@@ -17,12 +17,12 @@
   },
```

Comment:
<p>oki aqui es que iniciamos los breakout rooms correcto ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoLatinas/Intro-to-python/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>Sii! Estos ejercicios los dejaríamos para que los resuelvan elles soles.</p><p>Capaz que podemos armar un poll para que cada ver qué tal les fue, si obtuvieron los resultados correctos, aunque por ahí estaría bueno que alguien muestre cómo lo hizo y damos feedback de manera general.</p>

Reply:
Creo que es mejor que muestren lo que hicieron. No queremos marearlos con polls despues de los breakout rooms que ya es un mareo.

Reply:
Sii tenés razón!